doc/modules/kernel_ridge.rst

```diff
@@ -0,0 +1,55 @@
+.. _kernel_ridge:
+
+===========================
+Kernelized ridge regression
+===========================
+
+.. currentmodule:: sklearn.kernel_ridge
+
+Kernelized ridge regression implemented in the class :class:`KernelRidge`
+combines ridge regression (linear least squares plus l2-norm  regularization)
+with the kernel trick. It thus learns a linear function in the Reproducing
+kernel Hilbert space induced by the respective kernel. This corresponds to a
+non-linear function in the original space.
+
+The model learned by :class:`KernelRidge` is identical to support vector regression
+(:class:`SVR`). However, different loss functions are used (ridge versus
+epsilon-insensitive loss). In contrast to :class:`SVR`, fitting
+:class:`KernelRidge` can be done in closed-form and is typically faster for
+medium-sized datasets. On the other hand, the learned model is non-sparse and
+thus slower than SVR at prediction-time.
+
+Using :class:`KernelRidge` with a linear kernel can be advisable over 
+non-kernelized Ridge in situations when the number of feature dimensions D is
+considerably larger than the number of training datapoints N since the
+computational cost of solving the dual (kernelized) problem is :math:`O(N^3)`
+while the standard Ridge requires :math:`O(D^3)`.
```

Comment:
Indeed you can remove this paragraph since this is already taken care of in `Ridge`.